### PR TITLE
contribute an esp32-wroom-32d case design

### PR DESCRIPTION
**Pull request by knaak**
I am happy to contribute a ESP32-WROOM-32D outdoor case design for homeassistant-glow which is designed to house and protect an ESP32-WROOM-32D based Home Assistant Glow project from the elements.

### CODE REVIEW

File: docs/docs/reference/cases.mdx [hunk 1]
```diff
@@ -33,3 +33,20 @@ This case consists of 2 parts, a top and bottom where you can slide the parts ov
 | ---- | ----------- | -- |
 | ESP32S - Top | Top part of the case for the ESP32S | [Download](/cases/ESP32S/ESP32_top_side_v1.stl)
 | ESP32S - Bottom | Bottom part of the case for the ESP32S | [Download](/cases/ESP32S/ESP32_bottom_side_v1.stl)
+
```
Comment (klaasnicolaas) [inline docs/docs/reference/cases.mdx:36]:
```suggestion
```

File: docs/docs/reference/cases.mdx [hunk 2]
```diff
@@ -33,3 +33,20 @@ This case consists of 2 parts, a top and bottom where you can slide the parts ov
...
+
+## ESP32-WROOM-32D
+
+<p align="left">
+  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_inner_view.jpg').default} />
+  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_assembled_view.jpg').default} />
+</p>
+
+This case consists of 2 parts, a top and bottom where you can friction fit the pieces over each other and is designed to be used in both an indoor and outdoor environment. The LED status light is designed to slide into the housing and form a relatively tight seal.  The USB cable is designed to slide up to provide some protection to water intrusion.
```
Comment (klaasnicolaas) [inline docs/docs/reference/cases.mdx:45]:
```suggestion
This case consists of 2 parts, a top and bottom where you can friction fit the pieces over each other and is designed to be used in both an indoor and outdoor environment. The LED status light is designed to slide into the housing and form a relatively tight seal. The USB cable is designed to slide up to provide some protection to water intrusion.
```

File: docs/docs/reference/cases.mdx [hunk 3]
```diff
@@ -33,3 +33,20 @@ This case consists of 2 parts, a top and bottom where you can slide the parts ov
...
+
+If you are familiar with OnShape, you can find the model by searching for \"home-assistant-glow esp32-wroom-case\" where you can make a copy and modify it to your specifications.
+
+| Name | Description | ⬇️ |
+| ---- | ----------- | -- |
+| ESP32S - Top | Top part of the case for the ESP32D | [Download](/cases/ESP32-WROOM-32D/ESP32-WROOM-32D_top_case.stl)
```
Comment (klaasnicolaas) [inline docs/docs/reference/cases.mdx:51]:
```suggestion
| ESP32 WROOM - Top | Top part of the case for the ESP32D | [Download](/cases/ESP32-WROOM-32D/ESP32-WROOM-32D_top_case.stl)
```

File: docs/docs/reference/cases.mdx [hunk 4]
```diff
@@ -33,3 +33,20 @@ This case consists of 2 parts, a top and bottom where you can slide the parts ov
...
+| ESP32S - Bottom | Bottom part of the case for the ESP32D | [Download](/cases/ESP32-WROOM-32D/ESP32-WROOM-32D_bottom_case.stl)
```
Comment (klaasnicolaas) [inline docs/docs/reference/cases.mdx:52]:
```suggestion
| ESP32 WROOM - Bottom | Bottom part of the case for the ESP32D | [Download](/cases/ESP32-WROOM-32D/ESP32-WROOM-32D_bottom_case.stl)
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: docs/docs/reference/cases.mdx [hunk 5]
```diff
@@ -33,3 +33,19 @@ This case consists of 2 parts, a top and bottom where you can slide the parts ov
...
+## ESP32-WROOM-32D
+
+<p align="left">
+  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_inner_view.jpg').default} />
+  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_assembled_view.jpg').default} />
+</p>
+
+This case consists of 2 parts, a top and bottom where you can friction fit the pieces over each other and is designed to be used in both an indoor and outdoor environment. The LED status light is designed to slide into the housing and form a relatively tight seal. The USB cable is designed to slide up to provide some protection to water intrusion.
+
+If you are familiar with OnShape, you can find the model by searching for \"home-assistant-glow esp32-wroom-case\" where you can make a copy and modify it to your specifications.
```
[the remark this answers is not in the document]
Comment (knaak) [reply docs/docs/reference/cases.mdx]: Yes, in created a share url and added it which should be durable and added it as an option.  I am not exactly sure how mdx files are used, so please check that I created a link successfully.

File: docs/docs/reference/cases.mdx [hunk 6]
```diff
@@ -33,3 +33,19 @@ This case consists of 2 parts, a top and bottom where you can slide the parts ov
...
+## ESP32-WROOM-32D
+
+<p align="left">
+  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_inner_view.jpg').default} />
+  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_assembled_view.jpg').default} />
+</p>
+
+This case consists of 2 parts, a top and bottom where you can friction fit the pieces over each other and is designed to be used in both an indoor and outdoor environment. The LED status light is designed to slide into the housing and form a relatively tight seal. The USB cable is designed to slide up to provide some protection to water intrusion.
+
+If you are familiar with OnShape, you can find the model by searching for \"home-assistant-glow esp32-wroom-case\" where you can make a copy and modify it to your specifications, or by using this link [to the model.](https://cad.onshape.com/documents/28a914bca0ab451bdef13eb6/w/61ae11d2b0e90a508761a159/e/b81866b90bb3011f3cadb1f2).  An account may be required.
```
Comment (klaasnicolaas) [inline docs/docs/reference/cases.mdx:46]:
```suggestion
If you're familiar with OnShape, you can access the model using [this link](https://cad.onshape.com/documents/28a914bca0ab451bdef13eb6/w/61ae11d2b0e90a508761a159/e/b81866b90bb3011f3cadb1f2). Simply make a copy and customize it according to your specifications. Please note that having an account might be necessary to make adjustments.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**